src/components/button/Styles.scss

```diff
@@ -14,4 +19,14 @@
   &--inactive{
     background-color: $elephant-paw;
   }
```

Comment:
Move the rules for images contained inside the button here and delete the id from the component and from the css.